Opti/OptiADMM.m

```diff
@@ -222,7 +218,8 @@ function initialize(this,x0)
                     this.xopt=this.CG.xopt;
                 end
             else
-                this.xopt=this.solver(this.zn,this.rho_n, this.xopt);
+                zn = this.yn{n}+this.wn{n};
```

Comment:
I think that it should be zn = this.yn+this.wn;  (the solver requires all the n)

Reply:
Bien vu!
But there is no + for cells...